client/lib/plans/features-list.js

```diff
@@ -1073,14 +1080,20 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_BACKUP_DAILY_V2 ]: {
 		getSlug: () => constants.FEATURE_BACKUP_DAILY_V2,
-		getTitle: () => i18n.translate( 'Automated daily site backups' ),
+		getTitle: () =>
+			POPULAR_PRODUCTS_OFFERING_VARIANT === abtest( POPULAR_PRODUCTS_OFFERING_AB_TEST )
+				? i18n.translate( 'Automated daily backups (off-site)' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Daily Off-site Backups' )` *ES Score: 6*